web/docs/developer-guides/build-from-source.md

```diff
@@ -37,7 +37,7 @@ Every [release](https://github.com/tenzir/tenzir/releases) of Tenzir includes an
 |✓|[Apache Arrow](https://arrow.apache.org)|>= 8.0.0|Required for in-memory data representation. Must be built with Compute, Zstd and Parquet enabled.|
 |✓|[re2](https://github.com/google/re2)||Required for regular expressione evaluation.|
 |✓|[yaml-cpp](https://github.com/jbeder/yaml-cpp)|>= 0.6.2|Required for reading YAML configuration files.|
-|✓|[simdjson](https://github.com/simdjson/simdjson)|>= 3.1.0|Required for high-performance JSON parsing. (Bundled as submodule.)|
+|✓|[simdjson](https://github.com/simdjson/simdjson)|>= 3.2.0|Required for high-performance JSON parsing. (Bundled as submodule.)|
```

Comment:
The CMake feels like 3.1 is the minimum.

Reply:
Ah right, I forgot to undo this change. I made it slightly more lenient after @tobim's comment so it's 3.1+ rather than 3.2+; before this PR it was 3.1 exactly, which caused my local 3.2 installation of simdjson to always be ignored at the cost of additional compile time.

Reply:
Done and fixed.